website/data/docs-nav-data-hidden.json

```diff
@@ -1,32 +0,0 @@
-[
```

Comment:
This `*-nav-data-hidden.json` file was part of the temporary solution, so removing them. These nav-data items have been added to the proper `nav-data` file, with the `"hidden"` prop in place to reflect current state.